<ui version="4.0" >
 <class>DlgPrefMidiBindingsDlg</class>
 <widget class="QWidget" name="DlgPrefMidiBindingsDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="labelDeviceName" >
     <property name="font" >
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>Your Device Name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QPushButton" name="btnActivateDevice" >
     <property name="text" >
      <string>Activate</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" >
    <spacer name="horizontalSpacer" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>303</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="3" >
    <widget class="QToolBox" name="toolBox" >
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="page" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>514</width>
        <height>306</height>
       </rect>
      </property>
      <attribute name="label" >
       <string>MIDI Input</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2" >
       <item row="0" column="0" >
        <widget class="QPushButton" name="btnSingleLearn" >
         <property name="styleSheet" >
          <string notr="true" />
         </property>
         <property name="text" >
          <string>MIDI Learn</string>
         </property>
         <property name="checkable" >
          <bool>true</bool>
         </property>
         <property name="checked" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QPushButton" name="btnGroupLearn" >
         <property name="styleSheet" >
          <string notr="true" />
         </property>
         <property name="text" >
          <string>Group Learn</string>
         </property>
         <property name="checkable" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2" >
        <widget class="QTableView" name="m_pInputMappingTableView" />
       </item>
       <item row="2" column="0" colspan="2" >
        <widget class="QGroupBox" name="groupBoxInputManagement" >
         <property name="minimumSize" >
          <size>
           <width>300</width>
           <height>100</height>
          </size>
         </property>
         <property name="title" >
          <string>Controls</string>
         </property>
         <layout class="QGridLayout" name="_2" >
          <item row="0" column="0" >
           <widget class="QPushButton" name="btnAddBinding" >
            <property name="styleSheet" >
             <string notr="true" />
            </property>
            <property name="text" >
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3" >
           <widget class="QPushButton" name="btnExportXML" >
            <property name="styleSheet" >
             <string notr="true" />
            </property>
            <property name="text" >
             <string>Save to XML</string>
            </property>
           </widget>
          </item>
          <item row="0" column="5" >
           <widget class="QPushButton" name="btnImportXML" >
            <property name="styleSheet" >
             <string notr="true" />
            </property>
            <property name="text" >
             <string>Load from XML</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QPushButton" name="btnRemoveBinding" >
            <property name="styleSheet" >
             <string notr="true" />
            </property>
            <property name="text" >
             <string>Remove</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2" >
           <widget class="QPushButton" name="btnClearBindings" >
            <property name="autoFillBackground" >
             <bool>false</bool>
            </property>
            <property name="styleSheet" >
             <string notr="true" />
            </property>
            <property name="text" >
             <string>Clear All</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>514</width>
        <height>306</height>
       </rect>
      </property>
      <attribute name="label" >
       <string>MIDI Output</string>
      </attribute>
      <widget class="QLabel" name="label_2" >
       <property name="geometry" >
        <rect>
         <x>80</x>
         <y>0</y>
         <width>361</width>
         <height>321</height>
        </rect>
       </property>
       <property name="font" >
        <font>
         <family>Courier New</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text" >
        <string>                      ,;-. 
                    ,((--). 
                   /         
                  |          | 
                  |          | 
                 (,'"`.  ,'"`.) 
                 :     /     ; 
                 `.o  ,'`.  o,' 
                 (|`>'`--'`&lt;'|)            ,-, 
     ,.           |/        |         ,-./ / 
   _ | ,-.       (          )        | `-'`--. 
  ( `' (_/|__         (o   /       ,-'     ,-' 
   ;         )    ,|`.  - ,'|.      `-.   )  
   | (    ,-'   _/ `-.`""',-' ---.   /      ; 
   |     |   ,-'    /  /   |   |--/       | 
   |     |_,|    / /  /   /   |          | 
   |     `     |                /        ,' 
   |           |              | /      _,' 
   :           ,              `/------' 
    `-.___,---')                `. 
             ,'                    
            /                       
           :                        : 
           |                      _,| 
            --.___         __,--' ; 
             `.    `"""""""'      ,' 
               |                  | 
               |      .____,      | 
               |         |        | 
               |         |        | 
               |         |        | 
               |         |        | 
               |         |        | 
     -hrr-     |         |        | 
               |         |        | 
               |-._____,-|-.____,-| 
               |_        |_       | 
             ,'  `------'| `-----'  
            /           _|_          
            `--._____,-'   `-.___,-' 
</string>
       </property>
       <property name="scaledContents" >
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_3" >
       <property name="geometry" >
        <rect>
         <x>360</x>
         <y>280</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text" >
        <string>Mmmmm, MIDI....</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
